src/main/scala/com/gu/zuora/model/response/Responses.scala

```diff
@@ -32,7 +32,7 @@ object ZuoraErrorResponse {
 }
 
 case class ZuoraErrorResponse(success: Boolean, errors: List[ZuoraError])
-  extends Throwable(errors.asJson.spaces2) with ZuoraResponse {
+    extends Throwable(errors.asJson.spaces2) with ZuoraResponse {
```

Comment:
I've no idea why this file gets edited by using `sbt compile`! it's really annoying

Reply:
It's Scalariform messing with your head!